presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix ``IS DISTINCT FROM`` for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve performance of :func:`array_join`.
+* Improve efficiency of :func:`ST_Buffer`. The new implementation produces fewer buffer points on rounded corners, which will produce very similar but different results. It also better handles buffering with small (<1e-9) distances.
+* Improve OOM killer log output to put memory heavy nodes and queries first.
+* Improve efficiency by using JTS instead of Esri for many geometrical operations and fix incorrect calculation of extreme points in certain cases (:issue:`14031`).
+* Add forms of :func:`approx_percentile` accepting an accuracy parameter.
+* Add a new session property ``aggregation_partitioning_merging_strategy`` to configure partition merging strategy when adding exchange around aggregation node.
```

Comment:
I still don't think this is very clear -- what is a partition merging strategy? How would a user decide when to set this property or what the appropriate setting should be?  

Or if it's not really user facing, maybe it doesn't need a release note.

Reply:
It's a little bit tricky (at least for me) to summarize the change of behavior in a short note. If it's doable, can we keep this note with a link to the PR where I have detailed description on the change of behavior?

Another option is to exclude this note as I can imagine only very sophisticated user will need to leverage it. By default it does not change the current behavior.

Reply:
ok added a link to the PR

Reply:
we can probably leave out the note then, but adding a link to a PR or issue is also an option 

you can link to an issue or pr using the following syntax as appropriate.
 ```
For issue #1234
:issue:`1234`

For pr #1234
:pr:`1234`
```

Reply:
yes added a link to the PR